src/auth/auth.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Body, Controller, Get, Post, Req, Res, UseGuards } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { Request, Response } from 'express';
+import { AuthService } from './auth.service';
+import { UserDTO } from './dto/user.dto';
+
+@Controller('auth')
+export class AuthController {
+    constructor(private authService: AuthService){}
+
+     // 회원가입
+    @Post('/sign-up') 
+    async signup(
+        @Req() req: Request, @Body() UserDTO: UserDTO): Promise<UserDTO>{
+            return await this.authService.registerUser(UserDTO);
+        } 
+
+    // 로그인
+    @Post('/sign-in')
+    async signin(@Body() userDTO: UserDTO, @Res() res: Response): Promise<any>{
+        const jwt = await this.authService.validateUser(userDTO);
+        res.setHeader('Authorization', 'Bearer '+jwt.accessToken);
```

Comment:
P4; 보통 req/resp로 짓습니다.
```suggestion
        resp.setHeader('Authorization', 'Bearer '+jwt.accessToken);
```